test/viewmodels/map/map_pin_view_model_integration_test.dart

```diff
@@ -118,6 +141,60 @@ void main() {
 
         expect(pinListTuple, unorderedEquals(expectedPinsTuple));
       });
+
+      testWidgets("callback function is created as expected", (tester) async {
```

Comment:
This test does not fit in this file. `testWidgets` tests the UI, when here you want to test a view model. I think this test should be moved to one of the `views/pages/map` test suite.

Reply:
Fixed with [5179bf4](https://github.com/ProximaEPFL/proxima/pull/335/commits/5179bf4d69d67e32eea422348a488a327aa482dd)